central/signatureintegration/datastore/validate.go

```diff
@@ -64,3 +63,34 @@ func validateCosignVerification(config *storage.CosignPublicKeyVerification) err
 
 	return multiErr
 }
+
+func validateCosignCertificateVerification(configs []*storage.CosignCertificateVerification) error {
+	var multiErr error
+
+	for _, config := range configs {
+		if config.GetCertificateIdentity() == "" {
+			multiErr = multierror.Append(multiErr, errors.New("certificate identity should be filled"))
+		}
+
+		if _, err := regexp.Compile(config.GetCertificateIdentity()); err != nil {
```

Comment:
What's `GetCertificateIdentity`?

Reply:
Per [spec](https://github.com/sigstore/fulcio/blob/98929593d42be8d32aec6df8b8ebcbdb8c3b424a/docs/certificate-specification.md) the fulcio certificates [have two new OIDs](https://github.com/sigstore/fulcio/blob/main/docs/oid-info.md#terminology) which are used to specify the identity with which the certificate was issued for (i.e. the email of the entity requesting the cert) and the issuer that issued the certificate (i.e. your OIDC provider).